package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint .",
     "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml .",
     "test": "npm run test:unit",
-    "test:unit": "jest test/*.spec.js --silent",
+    "test:unit": "jest test/spec/ --silent",
```

Comment:
- how about merge `test` and `test:unit`. seems doesn't need both?
- maybe have another one for ci `test:report: npm run test -- --ci ...`

Reply:
Good idea. I had a feeling we'd eventually add `e2e` tests within this repo, but for now we should group these together.